packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -284,6 +300,10 @@ const Breakouts = WebexPlugin.extend({
     });
 
     this.breakouts.set(Object.values(breakouts));
+
+    if (this.allowBackToMain && this.getMainSession().requested) {
```

Comment:
I need to check `allowBackToMain` before calling `getMainSession`, otherwise `getMainSession` will throw an exception. Because if `allowBackToMain` is false, then there is no `main session`.